processors/base/helpers/__init__.py

```diff
@@ -86,6 +86,10 @@ def clean_identifiers(identifiers):
     }
     result = {}
     for key, value in identifiers.items():
+        try:
+            value = re.sub(r'\s', '', value)
+        except TypeError:
+            pass
```

Comment:
I like the idea, but unfortunately there're a few identifiers with whitespace (e.g. http://www.takedaclinicaltrials.com/browse/summary/AD-4833_409%20(PHARMO)?protocol_id=#overview), so we can't remove them all.